seumich/templates/seumich/student_list_partial.html

```diff
@@ -4,11 +4,7 @@
 {% cache settings.CACHE_TTL student_list_partial request.get_full_path %}
 <script src='{% static 'seumich/sort_table.js' %}'></script>
 
-<h1 class="sub-header">{{ studentListHeader }}</h1>
-<hr class="main-no-margin-top"/>
-{% if students %}
-    <h2 class="list-header">Students</h2>
-{% endif %}
+<h2 class="list-header">Students</h2>
```

Comment:
student_list_partial is used by 4 templates as identified on 
https://github.com/tl-its-umich-edu/student_explorer/pull/160#issuecomment-538115096

Just moving these lines into advisor_detail will only fix 1/4 of the places they are used. 

I think we should just leave everything in this file as-is and not add it into advisor_detail. It looks like it would end up the same as it is now.

Reply:
I'm not sure I agree, but perhaps I am missing something. It seems like in all cases it would be useful to have "Students" appear above a list of students, and that the name of the advisor (and the hr tag) should only be presented in relevant cases, i.e. `advisor_detail`. For cohort and class, the header should say something else. Thoughts?

Reply:
(I'm just learning how these templates are organized too, just want to make sure any fix for this is consistent across everything that touches this template and doesn't cause any regressions)

Reply:
Okay, I hadn't realized about the `studentListHeader` changing depending on context. I appreciate your attention to this. Let me experiment; I think what you're suggesting will work.

Reply:
@jonespm, take a look at [this solution](https://github.com/tl-its-umich-edu/student_explorer/pull/164/commits/8382badeab1b112e6bc96d9012f461e79d97f0ea). I moved the `studentListHeader` and `hr` tag back to `student_list_partial` and then I added those also to `advisors_without_students`. I thought it would be better to keep the logic of what is presented to advisors in `advisor_detail`, since we wouldn't want `advisor_without_students` to show up in any of the other cases.

Reply:
@jonespm, just following up here, I want to make sure I'm addressing your concern.

Reply:
I didn't see you were waiting for me, just click "request review" up on the top there and it should ping me again. I'll look at this this morning.

Reply:
Yeah, sorry, I need to get better at that. I figured you'd get a notification when I commented or mentioned you.